test/appium/views/chat_view.py

```diff
@@ -447,6 +460,10 @@ def chat_element_by_text(self, text):
         self.driver.info("Looking for a message by text: '%s'" % text)
         return ChatElementByText(self.driver, text)
 
+    def chat_element_by_text_part(self, text):
```

Comment:
This method is redundant because of two things:
1) It searches for text part (text = 'left group chat') which could be done with generic find_text_part() method
2) It does not confirm whether *user_name* (username of who left the chat) is present.

I'd go with the way as test_group_chat_system_messages test does for system message

Reply:
Well, I thought about it - but I don't think that we need especially grab a username for this particular check. For this we need to go to Profile -> Grab username especial for this part. For leaving group chat IMO it is redundant.


According to find_text_part()  - seems I missed this method, thanks!